inc/saineighbor.h

```diff
@@ -123,13 +123,14 @@ typedef enum _sai_neighbor_entry_attr_t
      *
      * @type sai_uint32_t
      * @flags CREATE_AND_SET
-     * @default 0
+     * @default internal
      */
     SAI_NEIGHBOR_ENTRY_ATTR_ENCAP_INDEX,
 
     /**
      * @brief Encapsulation index is imposed
      *
+     * This attribute is deprecated
```

Comment:
there is a special flag that specifies that attribute is depreacated

Reply:
Thanks, Made that change